### PR TITLE
Vue style guide

### PR DESCRIPTION
**Pull request by KlamChowder1**
- fixed some more UI stuff and cleaned up more code to adhere to proper vue style guides https://vuejs.org/v2/style-guide/

### CODE REVIEW

File: frontend/components/admin/UserForm.vue [hunk 1]
```diff
@@ -52,8 +65,14 @@ export default {
     isFormValid: false,
     name: '',
     username: '',
-    password: ''
+    password: '',
+    passwordConfirm: ''
   }),
+  computed: {
+    samePassword() {
```
Comment (asim-shrestha) [inline frontend/components/admin/UserForm.vue:72]: Looks like you meant to use this in the text field rules. Currently not in use

File: frontend/components/pipeline/PipelineResults.vue [hunk 2]
```diff
@@ -13,10 +13,27 @@
...
+          delete
+        />
+        <v-icon-btn
+          v-if="deleteMode"
+          @click="clearDelete"
+          :color="toolbarIconColor"
+          close
+        />
+        <v-icon-btn v-if="deleteMode" @click="saveDelete"
+        :color="toolbarIconColor" <<<<<<< HEAD save ======= deleteEmpty >>>>>>>
```
Comment (asim-shrestha) [inline frontend/components/pipeline/PipelineResults.vue:35]: Merge conflict remnants `<<<<<<< HEAD save ======= deleteEmpty >>>>>>>`

File: frontend/components/admin/UserList.vue [hunk 3]
```diff
@@ -19,7 +19,7 @@
       sort-by="name"
       :sort-desc="false"
     >
-      <template v-slot:item.ae_title="{ item }">
+      <template v-slot:[`item.ae_title`]="{ item }">
```
Comment (asim-shrestha) [inline frontend/components/admin/UserList.vue:22]: Is this from the style guide? How come we do this?

File: frontend/components/pipeline/PipelineResults.vue (hunk 2 above)
Comment (KlamChowder1) [reply frontend/components/pipeline/PipelineResults.vue:35]: rip coding at 3am

File: frontend/components/admin/UserList.vue (hunk 3 above)
Comment (KlamChowder1) [reply frontend/components/admin/UserList.vue:22]: this was moreso for ESlint complaining and not vue styles. could revert it back
https://stackoverflow.com/questions/61344980/v-slot-directive-doesnt-support-any-modifier

Comment (asim-shrestha) [reply frontend/components/admin/UserList.vue:22]: Gotcha, should leave it then

File: frontend/components/pipeline/PipelineResults.vue (hunk 2 above)
Comment (asim-shrestha) [reply frontend/components/pipeline/PipelineResults.vue]: LOL

**Review by awtkns — APPROVED** [review]

**Pull request merged**